utils/watchcat/Makefile

```diff
@@ -41,6 +41,8 @@ define Package/watchcat/install
 	$(INSTALL_BIN) ./files/watchcat.sh $(1)/usr/bin/watchcat.sh
 	$(INSTALL_DIR) $(1)/etc/uci-defaults
 	$(INSTALL_BIN) ./files/uci_defaults_watchcat $(1)/etc/uci-defaults/50-watchcat
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_DATA) ./files/watchcat.config $(1)/etc/config/watchcat
```

Comment:
why do we need a uci default script for the configuration? If we have our own file then we can write it directly.

Reply:
done that